test/kickstart-templates/includes/main-ostreecontainer.cfg

```diff
@@ -1,22 +1,33 @@
-# Enable insecure registry access from the hypervisor to allow bootc image pull.
-# See https://github.com/containers/bootc/blob/main/docs/src/registries-and-offline.md#insecure-registries
+# Configure support of pulling images both from a local mirror and remote registries.
+# Note: skopeo falls back to 'docker.io/library' prefix when specifying unqualified
+# image names. These need to be redirected to the local mirror location.
 %pre-install --log=/dev/console --erroronfail
 
 mkdir -p /etc/containers/registries.conf.d
-cat > /etc/containers/registries.conf.d/998-microshift-bootc-registry.conf <<EOF
+# Enable insecure registry access from the hypervisor to allow local image pull.
+# See https://github.com/containers/bootc/blob/main/docs/src/registries-and-offline.md#insecure-registries
+cat > /etc/containers/registries.conf.d/998-microshift-bootc-registry.conf <<'EOF'
 [[registry]]
+    prefix = "docker.io/library"
```

Comment:
@pmtk , I created the [USHIFT-5083](https://issues.redhat.com/browse/USHIFT-5083) follow-up as discussed to make sure we always use qualified image paths.

Reply:
Thanks! I totally forgot about that one